docs/DeveloperGuide.md

```diff
@@ -723,27 +723,112 @@ testers are expected to do more *exploratory* testing.
 
     1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-    1. Re-launch the app by double-clicking the jar file.<br>
+    1. Re-launch the app by double-clicking the jar file.
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+
+
+### Adding a company
+
+1. Adding a company
+
+    1. Prerequisites: None
+
+    1. Test case: `add -n ABC -e abc@abc.com -t XYZ`
+       Expected: Contact with name `ABC`, email `abc@abc.com`, tag `XYZ` is added to the bottom of the list. Details of the added contact shown in the status message. Timestamp in the status bar is updated.
+
+    1. Test case: `add -e abc@abc.com -t XYZ`
+       Expected: No company is added. Error details shown in the status message. Status bar remains the same.
+
+    1. Other incorrect delete commands to try: `add`, `add -n ABC -t XYZ`, `...` 
+       Expected: Similar to previous.
 
 ### Deleting a company
 
 1. Deleting a company while all companies are being shown
 
     1. Prerequisites: List all companies using the `list` command. Multiple companies in the list.
 
-    1. Test case: `delete 1`<br>
+    1. Test case: `delete 1`
        Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
 
-    1. Test case: `delete 0`<br>
+    1. Test case: `delete 0`
```

Comment:
Is this removal of line break intentional? I just though that it might be better to have a line break here